djstripe/models/base.py

```diff
@@ -201,10 +201,13 @@ def _manipulate_stripe_object_hook(cls, data):
 
     @classmethod
     def _find_owner_account(cls, data):
+        from .account import Account
+        stripe_account = getattr(data, "stripe_account", None)
+        if stripe_account:
+            return Account.objects.get(id=stripe_account)
```

Comment:
This will break if the Account object is not in the db yet.

Reply:
good catch.

Reply:
https://github.com/dj-stripe/dj-stripe/pull/1235 is an attempt to provide a mechanism for avoiding that.